tests/stdlib/tcolordiff.nim

```diff
@@ -0,0 +1,258 @@
+import experimental/[diff, colordiff, colortext]
```

Comment:
Not a blocker, but I feel like this should use unit test.

Reply:
just like every single test out there, right, when unittest becomes usable we can use it